lib/clients/web/facets/chat.js

```diff
@@ -175,4 +176,21 @@ ChatFacet.prototype.unfurl = function unfurl(ts, channel, unfurls, opts, optCb)
   return this.makeAPICall('chat.unfurl', requiredArgs, opts, optCb);
 };
 
+/**
+ * Retrieve a permalink URL for a specific extant message.
+ * @see {@link https://api.slack.com/methods/chat.getPermalink|chat.getPermalink}
+ *
+ * @param {?} ts - A message's ts value, uniquely identifying it within a channel.
+ * @param {?} channel - The ID of the conversation or channel containing the message.
+ * @param {function=} optCb Optional callback, if not using promises.
+ */
+ChatFacet.prototype.getPermalink = function delete_(ts, channel, optCb) {
```

Comment:
the order of the params should be as listed on the [documentation page](https://api.slack.com/methods/chat.getPermalink). we've had plenty of [issues](#428) from people who find the SDK to not be consistent in this way and its caused frustration.